### ISSUE TITLE
Input still has "vi__character--selected" class after disabling input through inputProps

### ISSUE THREAD

**Issue by piotr-ponikowski-source** [andreaswilli/react-verification-input#136]
Hi!

I have case when input looks as active, hovewer it is not focused anymore. This can be replicated when disabling input through inputProps, for exmaple during validation:
```
<VerificationInput inputProps={{ disabled: isValidatingPin }} />
```
The issue is that disabling input causes input to de-focus, however onBlur is not called (using Chrome). 

If I understand correctly this callback is not invoked in such case: https://github.com/andreaswilli/react-verification-input/blob/master/src/index.js#L119C1-L122C15
and input still has "selected" class: https://github.com/andreaswilli/react-verification-input/blob/master/src/index.js#L73

I was wondering if there is some workaround to remove "vi__character--selected" class from input? I was thinking also about preparing PR with logic like below more or less:
```
useEffect(() => {
  if(inputProps.disabled != null) {
    setActive(!inputProps.disabled);
  }
}, [inputProps.disabled]);
```

What do you think? Many thanks for great lib and help!

**Comment by andreaswilli** [comment]
Hi, thanks for the report!
I was able to reproduce the issue.

Your suggestion seems like a valid option (not sure about the null check).
Another option might be to turn `isActive` into a function and use the focus state + disabled state to derive the result.

Let me know whether you'd like to work on this.

**Comment by piotrponikowski** [comment]
Hi, I will try to work on this: https://github.com/andreaswilli/react-verification-input/pull/137
PS. I changed account to personal one, sorry for confusion

**Comment by andreaswilli** [comment]
Released `4.1.2`
